app/models/enhancements/application.rb

```diff
@@ -1,12 +1,26 @@
 require "doorkeeper/orm/active_record/application"
 
 class ::Doorkeeper::Application < ActiveRecord::Base
+  include ActiveModel::ForbiddenAttributesProtection
+
   has_many :permissions, :dependent => :destroy
   has_many :supported_permissions, :dependent => :destroy
 
-
-  attr_accessible :name, :description, :uid, :secret, :redirect_uri, :home_uri
-  attr_accessible :supports_push_updates, role: :superadmin
+  # TODO: Remove this accessible attributes list once Signonotron has upgraded
```

Comment:
Thanks for the tip @evilstreak. I've discussed this with Dom, but for anybody else reading: Unfortunately after trying the approach taken for Whitehall, it turns out this particular case is different in that the offending `attr_accessible` call originates from a module included in Doorkeeper's Application model, which our own model seems to be a *reopening* of and therefore makes it "pretty gross" to solve